test/functional/test_framework/util.py

```diff
@@ -190,6 +190,11 @@ def check_json_precision():
     if satoshis != 2000000000000003:
         raise RuntimeError("JSON encode/decode loses precision")
 
+def EncodeDecimal(o):
+    if isinstance(o, Decimal):
+        return str(o)
+    raise TypeError(repr(o) + " is not JSON serializable")
```

Comment:
Any reason why this is duplicated from the identical function in authproxy?

Reply:
There were include issues running tests directly when I moved the authproxy usage into util. I can take another look.

Reply:
ok, you may leave it as is